operators/src/processing/rasterization/mod.rs

```diff
@@ -446,17 +454,16 @@ impl RasterQueryProcessor for DensityRasterizationQueryProcessor {
                                             .collect::<Vec<_>>(),
                                     )
                                 })
-                                .for_each_with(send, |send, val| {
-                                    debug_assert!(send.send(val).is_ok());
+                                .for_each(|res| {
+                                    let mut tile_data = shared_tile_data.lock().unwrap();
```

Comment:
This somehow feels wrong, since you basically sequentialize the access to the tile.

What about something like this?
https://play.rust-lang.org/?version=stable&mode=debug&edition=2021&gist=20989f8bc076d3683b4b4a670b74977d

Generally, we would need a benchmark for that to really assess it.

Reply:
Simplified in https://github.com/geo-engine/geoengine/pull/683/commits/ac01de8afac02cff861242474bd23b6c894569b9